crates/bevy_math/src/rects/irect.rs

```diff
@@ -68,9 +71,9 @@ impl IRect {
     ///
     /// ```rust
     /// # use bevy_math::{IRect, IVec2};
-    /// let r = IRect::from_center_size(IVec2::ZERO, IVec2::splat(2)); // w=2 h=2
-    /// assert_eq!(r.min, IVec2::splat(-1));
-    /// assert_eq!(r.max, IVec2::splat(1));
+    /// let r = IRect::from_center_size(IVec2::ZERO, IVec2::new(3, 2)); // w=2 h=2
+    /// assert_eq!(r.min, IVec2::new(-1, -1));
```

Comment:
I'm fine with the change to `(3, 2)`, but the other ones are literally the canonical use case for `splat()` so using `new(x, x)` instead just encourages bad practices.